File: Web/RMA-Web-Shell/projects/fincare/src/app/payment-manager/models/payment-pool-search-params.ts
import { PaymentTypeEnum } from "projects/shared-models-lib/src/lib/enums/payment-type-enum";
import { PaymentStatusEnum } from "../../shared/enum/payment-status-enum";

export class PaymentPoolSearchParams {
    startDate: string;
    endDate: string;
    paymentTypeId: PaymentTypeEnum;
    claimTypeId: number;
    paymentStatusId: PaymentStatusEnum;
    page: number;
    pageSize: number;
    orderBy: string;
    isAscending: boolean;
    query: string;    
    reAllocate: boolean;
    userLoggedIn: number;
    workPoolId: number;
    coidPaymentTypeId: number;
    pensionPaymentTypeId: number;
}

File: Service/FinCare/RMA.Service.FinCare.Contracts/Entities/Payments/PaymentPoolSearchParams.cs
namespace RMA.Service.FinCare.Contracts.Entities.Payments
{
    public class PaymentPoolSearchParams
    {
        public string StartDate { get; set; }
        public string EndDate { get; set; }
        public int PaymentTypeId { get; set; }
        public int ClaimTypeId { get; set; }
        public int PaymentStatusId { get; set; }
        public int Page { get; set; }
        public int PageSize { get; set; }
        public string OrderBy { get; set; }
        public bool IsAscending { get; set; }
        public string Query { get; set; }

        public bool ReAllocate { get; set; }
        public int UserLoggedIn { get; set; }
        public int WorkPoolId { get; set; }     
        public int CoidPaymentTypeId { get; set; }
        public int PensionPaymentTypeId { get; set; }
    }
}
